lightning-invoice/src/utils.rs

```diff
@@ -591,46 +594,6 @@ mod test {
 		match_invoice_routes(Some(5000), &nodes[0], scid_aliases);
 	}
 
-	#[test]
-	fn test_forwarding_info_not_assigned_channel_excluded_from_hints() {
```

Comment:
This test required that the `Channel::counterparty_forwarding_info` field wasn't assigned until the `ChannelUpdate` msg. The `Channel::counterparty_forwarding_info` field will now be assigned as soon as it's accepted by both parties.